adodb-loadbalancer.inc.php

```diff
@@ -233,7 +233,7 @@ public function getLoadBalancedConnection($type)
      * @return bool|ADOConnection
      * @throws Exception
      */
-    private function _getConnection($connection_id)
+    public function _getConnection($connection_id)
```

Comment:
So we have some internal unit tests that need direct access to these functions to test some complex situations that can't easily be done when they are private. I would expect normal developers to make use of these functions directly. Do you have any other suggestions on how to better handle this?

Reply:
I'm fine with making the method it public; all I'm saying is that its name then needs to be changed to remove the `_` prefix, which historically denotes internal methods in ADOdb.

Since it was private until now, the refactoring should be quite straightforward (I see only 2 references within adodb-loadbalancer.inc.php) and without consequences to the general public.